html/index.html

```diff
@@ -34,7 +34,7 @@
         <div class="container">
             <div class="icontop">
                 <a class="navbar-brand js-scroll-trigger" href="#page-top">
-                    <img src="assets/img/logo-cuadrado-transparente-invertido.png" /> HacktoberWeek
+                    <img src="assets/img/logo-cuadrado-transparente-invertido.png" alt="Hacktoberweek 2020 Aula Software Libre UCO" /> HacktoberWeek
```

Comment:
Alt attribute should reflect the content of the image, in this case, you must to explain that the image contains the logo